src/schema/settings.ts

```diff
@@ -53,6 +56,20 @@ interface GQLUpdateSettingsInput extends Partial<GQLSettings> {
 export const typeDefs = /* GraphQL */ `
   ${toGQLEnum(ChecklistViewState, 'ChecklistViewState')}
 
+  type SettingsFlagsPublic {
+    sidebarSquadExpanded: Boolean
+    sidebarCustomFeedsExpanded: Boolean
+    sidebarOtherExpanded: Boolean
+    sidebarResourcesExpanded: Boolean
+  }
```

Comment:
This can be https://github.com/dailydotdev/daily-api/pull/2368#discussion_r1822516038

Reply:
also ignore